src/main/java/dev/tycho/stonks/managers/DatabaseHelper.java

```diff
@@ -49,23 +51,26 @@ public DatabaseHelper(Stonks plugin, DatabaseManager databaseManager) {
   }
 
   public void createCompany(Player player, String companyName) {
+    //Prevent the player from spamming companies
     if (!player.isOp() && playerCompanyCooldown.containsKey(player.getUniqueId()) && (System.currentTimeMillis() - playerCompanyCooldown.get(player.getUniqueId())) < COMPANY_CREATION_COOLDOWN) {
-      sendMessage(player, "You cannot make a company for another " + Util.convertString(COMPANY_CREATION_COOLDOWN - (System.currentTimeMillis() - playerCompanyCooldown.get(player.getUniqueId()))));
+      player.sendMessage(ChatColor.RED + "You cannot make a company for another " +
+          Util.convertString(COMPANY_CREATION_COOLDOWN - (System.currentTimeMillis() - playerCompanyCooldown.get(player.getUniqueId()))));
       return;
     }
     Stonks.newChain()
         .async(() -> {
           if (companyName.length() > 32) {
-            sendMessage(player, "A company name cannot be longer than 32 characters!");
+            player.sendMessage(ChatColor.RED + "A company name can't be longer than 32 characters!");
             return;
           }
           try {
             if (databaseManager.getCompanyDao().companyExists(companyName)) {
-              sendMessage(player, "A company with that name already exists!");
+              player.sendMessage(ChatColor.RED + "A company with that name already exists!");
```

Comment:
Revert this

Reply:
https://github.com/schlatt-co/stonks/blob/a34c87a8a16fdce21ba0462044aedb07a35af01b/src/main/java/dev/tycho/stonks/managers/DatabaseHelper.java#L67